### ISSUE TITLE
Autocorrect problem with Rails/Presence in the case of a multiplication

### ISSUE THREAD

**Issue by theodugautier** [rubocop/rubocop#11005]
In my case (see below) the `Rails/Presence` cops does not correctly correct the case of a multiplication, which produces a Zeitwerk error.

--------

## Expected behavior

There should be a space between the parenthesis and the * or remove the parenthesis and add a space between the * and the float/integer.

## Actual behavior

I have this error : `C: [Correctable] Rails/Presence: Use variable.presence || price.to_f *(12.0) instead of if variable.present?`
and when I execute `rubocop -a`, it corrects : `annual_contribution.presence || month_price.to_f *(12.0)`

it gives me this error when I run `bin/rails zeitwerk:check` :
```bash
syntax error, unexpected *, expecting end-of-input

   87  class TestingClass
  123    def test_function
❯ 124      variable.presence || price.to_f *(12.0)
  125    end
  135  end

rails aborted!
```

## Steps to reproduce the problem

```ruby
class TestingClass
  def test_function
      if variable.present?
        variable
      else
        price.to_f * 12.0
      end
  end
end
```

and run `rubocop -a`

## RuboCop version

```
$ bundle exec rubocop -V
1.25.1 (using Parser 3.1.1.0, rubocop-ast 1.16.0, running on ruby 2.7.6 arm64-darwin20)
  - rubocop-rails 2.13.2
```
